src/data.cpp

```diff
@@ -1125,7 +1125,7 @@ FILE*tryopen(const char*name,const char*mode,bool allownull=false){
 	exit(EDATA);
 }
 
-FILE*_myfopen(files file, bool write){
```

Comment:
The `write` parameter is used below when `WITH_FMEMOPEN` is defined (on line 1142).

This is the case when compiling on Linux.

With this patch I get `src/data.cpp:1142:7: warning: the address of ‘ssize_t write(int, const void*, size_t)’ will never be NULL [-Waddress]` -- because there's a builtin that it happened to shadow, otherwise it would just fail.